build_docs.pl

```diff
@@ -24,9 +24,6 @@ BEGIN
 }
 
 use lib 'lib';
-use Proc::PID::File;
-die "$0 already running\n"
-    if Proc::PID::File->running( dir => '.run' );
 
```

Comment:
PID files are lock files, and lock files are trouble. Good riddance.

I usually try to do some tricky integration of the process list to avoid PID files, and thus stale locks. The output of `ps -eo comm` and `ps -eo command` are favorites of mine.